src/mutation-hoc.tsx

```diff
@@ -31,7 +31,7 @@ export function withMutation<
   if (typeof mapPropsToOptions !== 'function') mapPropsToOptions = () => options as MutationOpts;
 
   return (
-    WrappedComponent: React.ComponentType<TChildProps & TProps>,
+    WrappedComponent: React.ComponentType<TProps & TChildProps>,
```

Comment:
Switching the `TProps & TChildProps` order around has no technical merit - it just bugged me!